cloudera/assets/dashboards/cloudera_overview.json

```diff
@@ -0,0 +1,1367 @@
+                                        },
+                                        {
+                                            "name": "query4",
+                                            "data_source": "metrics",
+                                            "query": "avg:cloudera.host.cpu_nice_rate{$cloudera_hostname,$cloudera_cluster} by {cloudera_hostname,cloudera_cluster}"
+                                        },
+                                        {
+                                            "name": "query5",
+                                            "data_source": "metrics",
+                                            "query": "avg:cloudera.host.cpu_iowait_rate{$cloudera_hostname,$cloudera_cluster} by {cloudera_hostname,cloudera_cluster}"
+                                        },
+                                        {
+                                            "name": "query6",
+                                            "data_source": "metrics",
+                                            "query": "avg:cloudera.host.cpu_irq_rate{$cloudera_hostname,$cloudera_cluster} by {cloudera_hostname,cloudera_cluster}"
+                                        },
+                                        {
+                                            "name": "query7",
+                                            "data_source": "metrics",
+                                            "query": "avg:cloudera.host.cpu_soft_irq_rate{$cloudera_hostname,$cloudera_cluster} by {cloudera_hostname,cloudera_cluster}"
+                                        },
+                                        {
+                                            "name": "query8",
+                                            "data_source": "metrics",
+                                            "query": "avg:cloudera.host.cpu_steal_rate{$cloudera_hostname,$cloudera_cluster} by {cloudera_hostname,cloudera_cluster}"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "palette": "dog_classic",
+                                        "line_type": "solid",
+                                        "line_width": "normal"
+                                    },
+                                    "display_type": "line"
+                                }
+                            ]
+                        },
+                        "layout": {
+                            "x": 0,
+                            "y": 0,
+                            "width": 6,
+                            "height": 3
+                        }
+                    },
+                    {
+                        "id": 7039978264190320,
+                        "definition": {
+                            "title": "Roles CPU Usage (WIP)",
```

Comment:
is it for "work in progress"?

Reply:
Ah good catch! I originally had something else in the chart, and I marked WIP since I was making changes. I have since fixed the data, and I'll remove the WIP in the title.